modular_citadel/code/modules/reagents/reagent container/hypospraymkii.dm

```diff
@@ -112,23 +112,23 @@
 /obj/item/hypospray/mkii/attackby(obj/item/I, mob/living/user)
 	if((istype(I, /obj/item/reagent_containers/glass/bottle/vial) && vial != null))
 		if(!quickload)
-			to_chat(user, "<span class='warning'>\The [src] can not hold more than one vial!</span>")
+			to_chat(user, "<span class='warning'>[src] can not hold more than one vial!</span>")
 			return FALSE
 		unload_hypo(vial, user)
 	if((istype(I, /obj/item/reagent_containers/glass/bottle/vial)))
 		var/obj/item/reagent_containers/glass/bottle/vial/V = I
 		if(!is_type_in_list(V, allowed_containers))
-			to_chat(user, "<span class='notice'>\The [src] doesn't accept this [vial].</span>")
+			to_chat(user, "<span class='notice'>[src] doesn't accept this [vial].</span>")
```

Comment:
Since I can't find it anywhere, how does \the work. From what I understand, it's ignored if the word following it is a proper noun (that is, capitalized and not initiated with \improper) and allowed to go through otherwise. If improper nouns work as the most recent commit suggests, the line I'm marking would possibly read "The hypospray doesn't accept this the hypovial."